TiborKovari/harry-potter-react
# tailwind css added and formatted

## What does this implement/fix?
- Integrates Tailwind CSS into the project to streamline styling and enhance the overall visual experience.
- Implements dynamic navigation logic:
    - The Home link now dynamically adjusts its visibility based on the user's current route, ensuring logical and intuitive navigation.
- Ensures smooth transitions between pages, reinforcing a seamless user experience.

## How to test?
1. Set up the development environment:
- Run npm install to ensure all dependencies are installed.
- Start the development server by executing npm run dev.

2. Access the application:
- Open http://localhost:5173/ in your preferred web browser.

3. Perform functional testing:
- Home link visibility:
    -Navigate to the Homepage: Confirm that the Home link is hidden.
    -Navigate to any other page (e.g., Characters): Confirm that the Home link is visible.
- Navigation functionality:
    -Test navigating between different routes to ensure transitions are smooth and that no unexpected behaviors occur.

## Any known issues?
- None identified at this stage.
